doc/GettingStarted.md

```diff
@@ -208,10 +210,10 @@ Newer versions should work as well, older versions might work as well.
 
 | OS         | distribution/version known to work (*) | Comment                                                                    |
 |------------|----------------------------------------|----------------------------------------------------------------------------|
-| GNU/Linux  | Manjaro                                | Last checked in January 2023                                               ||
-| GNU/Linux  | Ubuntu 20.04 - 22.10                   | All versions in that range work. 20.04 needs CMake installed from Snap.    |
+| GNU/Linux  | Manjaro                                | Last checked in January 2023                                               |
+| GNU/Linux  | Ubuntu 20.04 - 24.04                   | All versions in that range work. 20.04 needs CMake installed from Snap.    |
 | GNU/Linux  | Ubuntu 18.04                           | Used with Intel PAC D5005 FPGA, custom toolchain needed                    |
```

Comment:
The custom tool chain is something like #684. I'll add a link here when merging that PR